tests/unit/test_charm.py

```diff
@@ -399,8 +344,36 @@ def test_db_relation(self):
             "database name should be set after relation joined",
         )
 
+    def test_add_admin_user(self):
+        self.harness.disable_hooks()
+        self.add_database_relations()
+
+        charm: DiscourseCharm = typing.cast(DiscourseCharm, self.harness.charm)
+        with patch.object(Container, "exec", return_value=MockExecProcess()) as exec_mock:
+            self.harness.container_pebble_ready("discourse")
+
+            email = "sample@email.com"
+            password = "somepassword"  # nosec
+            event = MagicMock()
+            event.params = {
+                "email": email,
+                "password": password,
+            }
+            charm._on_add_admin_user_action(event)
+
+        exec_mock.assert_any_call(
+            [
+                "bash",
+                "-c",
+                "./bin/bundle exec rake admin:create <<< $'sample@email.com\nsomepassword\nsomepassword\nY'",
```

Comment:
Would it be worth changing this to an f-string with the email and password defined earlier? E.g., `f"./bin/bundle exec rake admin:create <<< $'{email}\n{password}\n{password}\nY'"`